examples/passkey-svelte/README.md

```diff
@@ -1,38 +1,52 @@
-# sv
+# Passkey Authentication (Svelte)
 
-Everything you need to build a Svelte project, powered by [`sv`](https://github.com/sveltejs/cli).
+This example app demonstrates how to implement passkey authentication in a Svelte application using Jazz.
 
-## Creating a project
+## Features
 
-If you're seeing this, you've probably already done this step. Congrats!
+- Passkey-based authentication
+- User registration and login
+- Secure credential management
+- Integration with Jazz authentication system
 
-```bash
-# create a new project in the current directory
-npx sv create
+## Getting Started
 
-# create a new project in my-app
-npx sv create my-app
+1. Clone the repository:
+
+```sh
+git clone https://github.com/garden-co/jazz.git
 ```
 
-## Developing
+2. Install dependencies:
 
-Once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), start a development server:
+```sh
+pnpm install
+```
```

Comment:
Do we need pnpm build as well?

Reply:
Oh, no, you don't need to build and run dev... 😅  Been a busy day